### PR TITLE
kconfig: fixes for Zephyr integration

### PR DESCRIPTION
**Pull request by brgl**
Here are two Kconfig fixes for bugs I noticed when integrating LVGL's Kconfig with Zephyr's.

### CODE REVIEW

File: Kconfig [hunk 1]
```diff
@@ -839,22 +839,19 @@ menu "LVGL configuration"
 
     menu "3rd Party Libraries"
         config LV_USE_FS_STDIO
-            int "File system on top of stdio API"
-            default 0
+            bool "File system on top of stdio API"
```
Comment (kisvegabor) [inline Kconfig:842]: Here you need to set an ASCII character (i.e. a driver letter) on which LVGL will reach this file system.
Finally, a file path will look like `"S/my/folder/file.txt"`.

Comment (brgl) [reply Kconfig:842]: Right. Is this the same for the other two too? Can we just add a sub-option for the filesystem letter?

Comment (kisvegabor) [reply Kconfig:842]: > Is this the same for the other two too

Yes, same for the others too. 

> Can we just add a sub-option for the filesystem letter?

I'm ok with that. User can choose from ASCII upper case letters and `\0` to disable.

Comment (rzr) [reply Kconfig:842]: or set default prefix set to "" for path starting with separator "/"

Comment (kisvegabor) [reply Kconfig:842]: @rzr It's only a character, not a string :slightly_frowning_face: 
E.g. `'A'`

Comment (rzr) [reply Kconfig:842]: yes but my point that string would be more generic like "A:" (or "df0:" for amiga os ;) )

Comment (kisvegabor) [reply Kconfig:842]: Yes, it'd be really better but would be an API break. We can do it only in v9.

**Comment by kisvegabor** [comment]
@brgl 
I've fixed the anonymous choice in 71c739cc2 and `LV_FS_` related parts in abba1c351.

So I close this PR. Feel free to open a new PR is you find any other issues with Kconfig.

**Pull request closed without merging**